### PR TITLE
Add a sample for subject "allusers".

### PR DESCRIPTION
**Pull request by liminw**

### CODE REVIEW

File: samples/bookinfo/kube/istio-rbac-productpage-alluser.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  namespace: default
+spec:
+  rules:
+  - services: ["productpage.default.svc.cluster.local"]
+    methods: ["GET"]
+---
+apiVersion: "config.istio.io/v1alpha2"
+kind: ServiceRoleBinding
+metadata:
+  name: bind-productpager-viewer
```
Comment (frankbu) [inline samples/bookinfo/kube/istio-rbac-productpage-alluser.yaml]: bind-productpage-viewer

Comment (liminw) [reply samples/bookinfo/kube/istio-rbac-productpage-alluser.yaml]: Fixed. Thanks.

**Comment by liminw** [comment]
Ping for reviews/approval. This is a very small change.

**Comment by mandarjog** [comment]
Any tests?

**Comment by liminw** [comment]
Thanks Mandar! The tests was submitted with code change https://github.com/istio/istio/pull/3253. This PR is just adding a sample.

**Pull request merged**